user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: I-DOCSIS_Create_CCAP_CM_pair
+---
+
+# Creating a new CCAP/CM pair
+
+To create a new CCAP/CM pair:
+
+1. In DataMiner Cube, go to apps > Automation.
+
+1. In the pane on the left, select the script *EPM_I_DOCSIS_AddNewCcapCmPair*.
+
+1. In the lower right corner, click *Execute*.
+
+1. UI steps:
+
+   **General**
+
+   1. Define the element name.
+
+      The name of the collector element will consist of the defined element name with the suffix "_COLLECTOR".
+
+   1. Select one of the back-end DMAs.
+
+   **CCAP Details**
+
+   1. Select the desired CCAP protocol.
+
+   1. Select the desired protocol version. We recommend to be the production version.
+
+   1. Enter the IP Address.
+
+   1. Define the desired community string.
+
+   **View Details**
+
+   1. Choose the location where the the CCAP/CM pair will be added according to the EPM view structure.
+
+   **Interface Setting**
+
+   1. Define the Username and password with an account that can access the **FE DMA**.
+
```

Comment:
What is the last step in this procedure? Do you need to click Finish, Confirm, Save, ... ?